### PR TITLE
[SWPWA-155] Welcome message

### PR DESCRIPTION
**Pull request by dmitrijs-voronovs**
* implemented welcome message
* added `WelcomeMessage` component

For guest:
image

For logged-in user
image

### CODE REVIEW

File: src/app/component/WelcomeMessage/WelcomeMessage.style.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+:root {
+    --welcome-message-height: 0px;
```
Comment (Danpiel) [inline src/app/component/WelcomeMessage/WelcomeMessage.style.scss]: Codacy Issue found: [Unexpected unit (length-zero-no-unit)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4766232)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/app/component/WelcomeMessage/WelcomeMessage.component.js  4
- src/app/component/WelcomeMessage/WelcomeMessage.container.js  4
- src/app/store/Config/Config.reducer.js  1
- src/app/store/Config/Config.dispatcher.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4766232&bid=15887936)

**Comment by alfredsgenkins** [comment]
@lianastaskevica I do not like styles. What do you think?

**Comment by alfredsgenkins** [comment]
Closing, no intent to implement.

**Pull request closed without merging**